cli/azd/.vscode/cspell.yaml

```diff
@@ -1,22 +1,13 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It would be nice if there was an output to let the user know where the state is stored.

After azd command is run that uses the env, say, your env info is located here:

Reply:
In addition, if there's an error when saving to remote state will there be a printed error message?

Reply:
Yes, if there is any error saving the state to the remote environment storage an error will be presented to the user.

The remote state configuration is an opt-in feature where the user must specify the storage account and optional container name where the state will be stored.  There are various areas in `azd` where we manipulate the azd env vars. 

Maybe something should print out in the `azd env list` command when it is configured otherwise if feels like it would be a bit too much.

Reply:
Yeah, maybe when you say azd env list it could say if the env is remote...maybe it already does?

Reply:
Yes, it shows 2 new columns of local & remote so you can see if the environment is local only, remote or both.